### ISSUE TITLE
New Paper (Other): SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor

### ISSUE THREAD

**Issue by rdvelazquez** [greenelab/covid19-review#137]
Title: SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor

# General Information
## Please paste a link to the paper or a citation here:

Link: https://www.sciencedirect.com/science/article/pii/S0092867420302294

## What is the paper's [Manubot-style citation](https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations)?

Citation: [@doi:10.1016/j.cell.2020.02.052]

### Is this paper primarily relevant to Background or Pathogesis? 

- [x] Background
- [x] Pathogenesis
- [ ] Methods

## Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19

* Spike
* Cell Entry
* ACE2
* TMPRSS2
* Protease
* Sera

## Which areas of expertise are particularly relevant to the paper?

- [x] virology
- [ ] epidemiology
- [ ] biostatistics
- [x] immunology
- [x] pharmacology
- [ ] other:

**Comment by rdvelazquez** [comment]
I won't try to summarize this paper (mostly over my head) but I saw that it is used as a reference in the pathogenesis section for, "sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry"

and was wondering if someone with expertise in this are wanted to double check to make sure this statement couldn't be misinterpreted. It seems like the paper states it less conclusively:
- The highlights section of the paper states, "Antibodies against SARS-CoV spike may offer some protection against SARS-CoV-2"
- The graphical abstract lists SARS-CoV-2 Neutralization Efficiency by SARS Convalescent Sera as "Yes (Moderate)"
- and Figure 5 shows the values of the applicable experiment

**Comment by nilswellhausen** [comment]
Punchline: I think "sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry" is an overstatement.

Arguments: The paper at #97 shows that most cross-reactivity between the different SARS strains is caused by binding to the S protein outside of the important RBD. This suggests that cross-reactivity is often overinterpreted between different strains. The blocking of viral entry with the patient sera from SARS patients in Fig.5 is not really impressive for SARS-CoV-2, so I would definitely not use the word "effectively".

I would change that sentence to "sera from convalescent SARS-CoV patients might have some efficacy to cross-neutralize SARS-CoV-2-S-driven entry"

**Comment by cgreene** [comment]
I agree that it's an overstatement. @nilswellhausen : can you propose a modification via a pull request?

**Comment by nilswellhausen** [comment]
Yes did that #139
